xml/System.Windows.Controls/UserControl.xml

```diff
@@ -47,7 +47,7 @@
   
  ]]></format>
     </remarks>
-    <related type="ExternalDocumentation" href="https://go.microsoft.com/fwlink/?LinkID=160025">NumericUpDown Custom Control with Theme and UI Automation Support Sample</related>
+    <related type="ExternalDocumentation" href="https://github.com/dotnet/docs-desktop/tree/main/dotnet-desktop-guide/samples/snippets/csharp/VS_Snippets_Wpf/UserControlNumericUpDown/CSharp">NumericUpDown Custom Control with Theme and UI Automation Support Sample</related>
```

Comment:
This one should be deleted. We shouldn't deep link into the snippets area, only samples.